armi/cli/tests/test_runSuite.py

```diff
@@ -47,3 +58,26 @@ def test_showVersion(self):
 
         self.assertIn("armi", out.getvalue())
         self.assertIn(meta.__version__, out.getvalue())
+
+    @patch("armi.cli.ArmiCLI.executeCommand")
+    def test_run(self, mockExeCmd):
+        """Test the ArmiCLI.run method.
+
+        .. test:: The ARMI CLI's import run() method works.
+            :id: T_ARMI_CLI_CS2
+            :tests: R_ARMI_CLI_CS
+        """
+        correct = 0
+        acli = ArmiCLI()
+
+        mockExeCmd.return_value = correct
+
+        origout = sys.stdout
+        try:
+            out = io.StringIO()
+            sys.stdout = out
+            ret = acli.run()
+        finally:
+            sys.stdout = origout
```

Comment:
The test all looks good, but why did you include the try/finally statements? Shouldn't the test be ok to just do?
```python
correct = 0
acli = ArmiCLI()
mockExeCmd.return_value = correct
ret = acli.run()
self.assertEqual(ret, correct)
```

Reply:
Hmm... Yeah, that's a good cleanup.

The truth is, of course, that I was playing around with the test and the logging stuff is all leftover from previous iterations.

I'll remove them.

Reply:
fixed